tests/smartswitch/common/device_utils_dpu.py

```diff
@@ -8,7 +8,7 @@
 from tests.common.helpers.platform_api import chassis, module
 from tests.common.utilities import wait_until
 from tests.common.helpers.assertions import pytest_assert
-from pkg_resources import parse_version
+import re
```

Comment:
can you move this import together with all imports in the beginning.

Reply:
removed it as it is not needed now.